pkgs/applications/office/p3x-onenote/default.nix

```diff
@@ -0,0 +1,57 @@
+{ stdenv, appimageTools, autoPatchelfHook, desktop-file-utils
+  , fetchurl, runtimeShell }:
```

Comment:
jnit: comma usually start at the begging of the line for input attr sets
```suggestion
, fetchurl, runtimeShell }:
```

Reply:
thanks, I've upped it since runtimeShell is now removed